### ISSUE TITLE
CodeLens color contrast should be 4.5:1 in HC theme

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#144920]
Refs: https://github.com/microsoft/vscode/issues/126899

@daviddossett can you please double check if the CodeLens color contrast in HC themes (both light and dark is at least 4.5:1)

This issue comes back from the accessibility testing time from time to time so I would love if we are good here. Thanks!

**Comment by daviddossett** [comment]
Here's the current state of the HC CodeLens contrast:

## Dark HC
<img width="599" alt="CleanShot 2022-03-10 at 13 34 01@2x" src="https://user-images.githubusercontent.com/25163139/158221290-f17394b4-53b4-4708-88fb-0bd7035daaa5.png">

## Light HC
<img width="605" alt="CleanShot 2022-03-10 at 13 34 23@2x" src="https://user-images.githubusercontent.com/25163139/158221310-ac99b141-52c6-4fa9-bd02-d1d7ba0caf78.png">
